src/components/DrawerPage/DrawerPage.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { Drawer } from 'antd';
+import styled from 'styled-components';
+
+const StyledDrawer = styled(Drawer)`
+  .ant-drawer-content {
+    background-color: ${props => props.theme.colors.utils.background.mid};
+  }
+
+  .ant-drawer-body {
+    padding: 0;
+  }
+`;
+
+type DrawerPageProps = {
+  title: string;
+  children: JSX.Element | string;
+  OnClickComponent: React.ElementType;
```

Comment:
... and please, use ElementType instead of React.ElementType

Reply:
I prefer this way of accessing react types. If this has some disadvantages I have added this for discussion here:
https://www.notion.so/appsome/Style-guide-a0a4ac34799847febe7373123e0fa6ef
If we will have a few more items we can spend some time on it and create eslint prettier rules for that. I propose to not focus on such low priority items right now, but do it in a bulk later on. What do you think?

Reply:
Ok, but You actually did extraction already with { useState }. That's why I put this comment - to hold some minimum standard in our code base.